wfe/wfe_test.go

```diff
@@ -20,6 +20,8 @@ import (
 	"testing"
 	"time"
 
+	jose "gopkg.in/square/go-jose.v2"
```

Comment:
Is this a spurious diff? I don't see anything using the new `jose` alias.

Reply:
This got moved from a little lower. It is technically a spurious diff unless there is a good reason to move it. :-)

Reply:
Not a big deal either way! The `"jose"` alias was also added with the move and I don't think it's required.

Reply:
This feedback is too minor to block this PR on. I'm going to +1 and merge.